lm_eval/tasks/__init__.py

```diff
@@ -116,6 +117,13 @@
     "piqa": piqa.PiQA,
     "prost": prost.PROST,
     "mc_taco": mc_taco.MCTACO,
+    
+    "xnli_all_languages": xnli.XNLI_all,
```

Comment:
Each example in the `xnli_all` subset is structured as a nested dictionary containing the same premise translated across 15 languages, e.g.:
```
{
    'premise': {
        'ar': 'تعرف عندما ياتي والديهم ومن الصعب ان اخرجهم و الكثير من الاباء لديهم اماكن للذهاب و اشياء كهذه و الوقت متاخر في الليل لذا', 
        'bg': 'нали знаеш кога родителите им идват и е трудно да ги измъкнеш и много родители имат места да ходят и такива неща и е късно през нощта така че', 
        'de': 'Du weißt , wenn ihre Eltern kommen und es schwer ist , sie raus zu bekommen und viele Eltern haben Orte zu gehen und solche Dinge , und es ist spät in der Nacht , so dass es zu spät ist .', 
        'el': 'Ξέρεις πότε έρχονται οι γονείς τους και είναι δύσκολο να τους βγάλεις έξω και πολλοί γονείς έχουν μέρη να πάνε και τέτοια πράγματα και είναι αργά το βράδυ έτσι', 
        'en': "you know when their parents come and it 's hard to get t...
    }
}
```
Working with these sort of nested dicts is kind of annoying in the harness and there's currently no way to group a bunch of related taks (as we have here).
I've gone ahead and just flattend everything into individual tasks.

Reply:
Thank you!!